cross/build-android-rootfs.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
```

Comment:
To your points:

1. See [build-android-rootfs.sh:39](https://github.com/dotnet/coreclr/blob/master/cross/build-android-rootfs.sh#L39): you can tell the script to create the rootfs for arm or arm64 by specifying arm or arm64 as a command line parameter
2. Yes, both corefx and coreclr will compile for arm as well.
3. Yes, but you can still switch back to r13b by changing [build-android-rootfs.sh:2](https://github.com/dotnet/coreclr/blob/master/cross/build-android-rootfs.sh#L2). Or you could update the script to take the NDK version as a parameter to make things even more flexible.